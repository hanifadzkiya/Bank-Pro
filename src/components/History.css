#tableID {
    font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
    border-collapse: collapse;
    width: 100%;
}

#tableID td, #tableID th {
    border: 1px solid #ddd;
    padding: 8px;
}

#tableID tr:nth-child(even){background-color: #f2f2f2;}

#tableID tr:hover {background-color: #ddd;}

#tableID th {
    padding-top: 12px;
    padding-bottom: 12px;
    text-align: left;
    background-color: #4CAF50;
    color: white;
}