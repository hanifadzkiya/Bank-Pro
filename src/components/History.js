import React, { Component } from "react";

class History extends Component {
    render() {
        return (
            <div>
                <h2>History</h2>
                <p>Pada riwayat transaksi, pengguna dapat melihat daftar data berikut: waktu transaksi, jenis transaksi (debit/kredit),
                    jumlah transaksi, dan rekening terkait (jika ada).</p>
            </div>
        );
    }
}

export default History;