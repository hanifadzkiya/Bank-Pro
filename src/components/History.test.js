import React from "react";
import History from './History';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

import { shallow } from 'enzyme';

it('Render History', () => {
    const wrapper = shallow(<History />);
    expect(wrapper.contains(<h2>History</h2>)).toEqual(true);
    expect(wrapper.contains(<p>Pada riwayat transaksi, pengguna dapat melihat daftar data berikut: waktu transaksi, jenis transaksi (debit/kredit),
        jumlah transaksi, dan rekening terkait (jika ada).</p>)).toEqual(true);

    expect(wrapper).toMatchSnapshot();
})