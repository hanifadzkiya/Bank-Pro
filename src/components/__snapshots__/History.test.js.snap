// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render History 1`] = `ShallowWrapper {}`;
